src/app/modules/feedback/services/feedback.service.ts

```diff
@@ -182,6 +190,26 @@ export class FeedbackService {
     );
   }
 
+  showSnackbar(ticketUrl?: string): MatSnackBarRef<SnackbarComponent> {
```

Comment:
This method name and implementation is ambiguous. We should have two methods instead of one.
`showTicketSuccessMsg(url: string): MatSnackBarRef<SnackbarComponent>`
and
`showFeedbackSuccessMsg(): MatSnackBarRef<SnackbarComponent>`